kubernetes_state/metadata.csv

```diff
@@ -54,6 +54,10 @@ kubernetes_state.nodes.by_condition,gauge,,,,To sum by `condition` and `status`
 kubernetes_state.hpa.min_replicas,gauge,,,,Lower limit for the number of pods that can be set by the autoscaler,0,kubernetes,k8s_state.hpa.min_replicas
 kubernetes_state.hpa.max_replicas,gauge,,,,Upper limit for the number of pods that can be set by the autoscaler,0,kubernetes,k8s_state.hpa.max_replicas
 kubernetes_state.hpa.desired_replicas,gauge,,,,Desired number of replicas of pods managed by this autoscaler,0,kubernetes,k8s_state.hpa.desired_replicas
+kubernetes_state.pdb.pods_desired,gauge,,,,Minimum desired number of healthy pods,0,kubernetes,k8s_state.pdb.pods_desired
+kubernetes_state.pdb.disruptions_allowed,gauge,,,,Number of pod disruptions that are currently allowed,0,kubernetes,k8s_state.pdb.disruptions_allowed
+kubernetes_state.pdb.pods_healthy,gauge,,,,Current number of healthy pods,0,kubernetes,k8s_state.pdb.pods_healthy
+kubernetes_state.pdb.pods,gauge,,,,Total number of pods counted by this disruption budget,0,kubernetes,k8s_state.pdb.pods
```

Comment:
Would it make sense to rename it `.pdb.pods_total` for consistency with the other gauges, and being more explicit?

Reply:
I concur as it would also be consistent with what one sees when using `kubectl describe pdb ...`.